docs/contributing/development.md

```diff
@@ -11,22 +11,97 @@
 In order to develop new features or fix bugs etc. we need to set up a virtual
 environment and install the library locally.
 
-## Quickstart development
+## Development environment setup
 
-The fastest way to start with development is to use nox. If you don't have nox,
-you can use `pipx run nox` to run it without installing, or `pipx install nox`.
-If you don't have pipx (pip for applications), then you can install with
-`pip install pipx` (the only case were installing an application with regular
-pip is reasonable). If you use macOS, then pipx and nox are both in brew, use
-`brew install pipx nox`.
 
-To use, run `nox` without any arguments.  This will run the type check and unit
-test "sessions" (tasks) using your local (and active) Python version.
-Nox handles everything for you, including setting up a temporary virtual
-environment for each run.
+There are a few options for setting up a development environment; you can use Python's `venv`, use `conda`/`mamba`, or _not_
+manage one yourself and use `pipx` to run tests and build docs with `nox`.
 
-You can see all available sessions with `nox --list`:
+* If you're a Windows user, you'll likely want to set up an environment yourself with `conda`/`mamba`.
+* If you're working in a JupyterHub, you'll likely want to set up an environment yourself with `conda`/`mamba`.
+* If you're using an IDE like VS Code or PyCharm, you'll likely want to set up an environment yourself with `venv` or `conda`/`mamba`.
+* If you're using a plain text editor and don't know how to or want to manage a virtual environment, you'll likely want to start with `pipx`.
+
+=== "`venv`"
```

Comment:
Something I really don't like about the tabbed sections as they are now is that it's very difficult to see where the bottom is and regular instructions resume. I pushed a commit subtly changing the CSS and I think it makes a huge difference. What do you think?

## New

![image](https://github.com/user-attachments/assets/c769029f-629f-49c9-9cd4-64a6e44692f8)


## Current

![image](https://github.com/user-attachments/assets/77607ec9-ead3-4202-bd49-abb608af1a3a)

Reply:
Yes, I like having the outline; I'd be open to making the tabs and box even more obvious -- I might play around with it.

Reply:
I've updated it to match the tables in our API Docs. 

Now: 
![Uploading image.png…]()


API Docs:
![image](https://github.com/user-attachments/assets/9c52048f-addf-496b-9a0e-ebc0118968a7)

Reply:
Love it! :star_struck: